tests/basic/misc_test.go

```diff
@@ -1784,4 +1784,71 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		var testrailID = 87311294
+		StartTorpedoTest("VerifyNoPxRestartDueToPxPodStop", "Verify that px serivce remain up even if px pod got deleted ", nil, 0)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to setup, validate and teardown apps", func() {
+
+		// Get uptime for px service on each node
+		stepLog = "Getting PID of Px process before and after PX pods on all the nodes"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			processPid := make(map[string]string)
+			processCmd := "pidof px"
+			storageNode := node.GetStorageNodes()
+			storageLessNodes := node.GetStorageLessNodes()
+			var AllNodes []node.Node
+			AllNodes = append(storageNode, storageLessNodes...)
+			//Capturing PID pf PX before stopping PX pods
+			for _, nnode := range AllNodes {
+				output, _ := Inst().N.RunCommand(nnode, processCmd, node.ConnectionOpts{
+					Timeout:         30 * time.Second,
+					TimeBeforeRetry: 20 * time.Second,
+					Sudo:            true,
+				})
+				processPid[nnode.Id] = output
+			}
+			log.Infof(fmt.Sprintf("Process IDs for px before stopping portworx pod  %s", processPid))
+
+			//Deleting px pods from all the node
+			namespace, err := Inst().S.GetPortworxNamespace()
+			if err != nil {
+				log.FailOnError(fmt.Errorf("Portworx namespace  %s is not found: err %v", namespace, err), "No namespace found with Px pods")
+			}
+			err = DeletePXPods(namespace)
+			//Capturing PID pf PX after stopping PX pods
+			processPidPostRestart := make(map[string]string)
+			for _, nnode := range AllNodes {
+				err := Inst().V.WaitForPxPodsToBeUp(nnode)
+				if err != nil {
+					log.FailOnError(fmt.Errorf("PX POD is not up, we can not test further due to err %s", err), "PX POD is down on node %s", nnode.Name)
+				}
+				output, err := Inst().N.RunCommand(nnode, processCmd, node.ConnectionOpts{
+					Timeout:         20 * time.Second,
+					TimeBeforeRetry: 5 * time.Second,
+					Sudo:            true,
+				})
+				if err != nil {
+					log.FailOnError(fmt.Errorf("Fail to get process id of PX process"), "Fail to get process id of PX process in node %s due to  %s", nnode.Name, err)
+				}
+				processPidPostRestart[nnode.Id] = output
+			}
+			log.Infof(fmt.Sprintf("Process IDs for px after stopping portworx pod  %s", processPidPostRestart))
+			//Verify PID before and after for PX process
+			for nodeDetails, beforePID := range processPid {
+				afterPID, _ := processPidPostRestart[nodeDetails]
+				dash.VerifyFatal(beforePID == afterPID, true, fmt.Sprintf("Process ID of PX process before anfter PX pod restart is differentfor px pr"))
```

Comment:
you can give as below 
`dash.VerifyFatal(afterPID,beforePID, fmt.Sprintf("Process ID of PX process in the node [%s] is not matching after px pod restart", nodeDetails))`

Reply:
Using `dash.VerifyFatal` as shown above will print the actual (`afterPID`) and expected (`beforePID`) values in the logs. Currently, it will be printed as `actual: <true/false>, expected: true`.